frontend/src/lib/components/PayGateMini/PayGateMiniButton.tsx

```diff
@@ -0,0 +1,56 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { billingProductLogic } from 'scenes/billing/billingProductLogic'
+import { PlanComparisonModal } from 'scenes/billing/PlanComparison'
+import { urls } from 'scenes/urls'
+
+import { BillingProductV2Type, BillingV2FeatureType } from '~/types'
+
+export const PayGateMiniButton = ({
+    product,
+    gateVariant,
+    featureInfo,
+    onClick,
+}: {
+    product: BillingProductV2Type
+    featureInfo: BillingV2FeatureType
+    gateVariant: 'add-card' | 'contact-sales' | 'move-to-cloud'
+    onClick?: () => void
+}): JSX.Element => {
+    const { isPlanComparisonModalOpen } = useValues(billingProductLogic({ product }))
+    const { toggleIsPlanComparisonModalOpen } = useActions(billingProductLogic({ product }))
+
+    return (
+        <>
+            <LemonButton
+                to={
+                    gateVariant === 'contact-sales'
+                        ? `mailto:sales@posthog.com?subject=Inquiring about ${featureInfo.name}`
+                        : gateVariant === 'move-to-cloud'
```

Comment:
maybe move to cloud links to some docs page?

Reply:
it should link to this internal page, does it not for you?

<img width="1210" alt="image" src="https://github.com/PostHog/posthog/assets/18598166/638871a3-b9ce-4cdc-9507-8946fbe10abd">

Reply:
Oh, sorry, I completely misread this ternary and thought it went to undefined. Looks good 😅